src/main/resources/project/procedures/preamble.groovy

```diff
@@ -912,3 +922,227 @@ public createVnet(def vnetName, def subnetName, def vnetAddressSpace, def subnet
 
 }
 
+public class SQLOperations {
+    //Database Connection object.
+    def dbCon
+
+    SQLOperations(server, database, port, user, password)
+    {
+        if (!setDatabaseConnection(server, database, port, user, password))
+        {
+            println("Could not set database connection object")
+            System.exit(1)
+        }
+    }
+
+    def setDatabaseConnection(server, database, port, user, password){
+        try{
+            def connectionUrl = "jdbc:jtds:sqlserver://" + server + ".database.windows.net:" + port +
+                ";database=" + database +
+                ";encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30"
+
+            dbCon = Sql.newInstance( connectionUrl, user , password ,"net.sourceforge.jtds.jdbc.Driver" )
+            return true
+        }catch(Exception ex) {
+            println(ex.toString())
```

Comment:
Will be using the same exception handler. It will set error log also and mark procedure as failed.